ci/values.yaml

```diff
@@ -97,6 +103,15 @@ config:
       - key: service
         from_attribute: service_name
         action: upsert
+    transform:
```

Comment:
This is fine, but why not use a resource processor instead ? Seems like we are just trying to insert resource attributes, and don't have some complex transformation that requires transform processor ?

Reply:
Discussed offline, given the possibility that we will drop the resource processor in the future this seems more future proof to me.